dotnet/src/InternalUtilities/planning/PlannerInstrumentation.cs

```diff
@@ -29,9 +29,9 @@ public static class PlannerInstrumentation
 
     /// <summary>Invokes the supplied <paramref name="createPlanAsync"/> delegate, surrounded by logging and metrics.</summary>
```

Comment:
See Stephen's PR https://github.com/microsoft/semantic-kernel/pull/3763 where he makes this internal